platform/broadcom/sonic-platform-modules-accton/as7326-56x/utils/pddf_post_driver_install.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Please check the permission of this file.

Reply:
Done to update the permission of this file.